src/app/query/account/account.query.ts

```diff
@@ -0,0 +1,31 @@
+import { useQuery } from 'react-query';
+import { AccountDataResponse } from '@stacks/stacks-blockchain-api-types';
+
+import { useApi, Api } from '@app/store/common/api-clients.hooks';
+import { useSetAccountInfo } from '@app/store/accounts/account.hooks';
+
+const STALE_TIME = 5 * 60 * 1000;
```

Comment:
I feel like we have this in multiple places, should it be moved to a shared constant?

Reply:
Yes a default stale time as shared constant might be nice. Bringing another library just for that is not worth it.

Reply:
You can set the [default stale time with react-query](https://react-query.tanstack.com/reference/QueryClient#queryclientsetdefaultoptions), so there's no need for a default constant. 

But I want to highlight the point that we should seldom use a default. A stale time should consider:
- How expensive the given API request is
- How sensitive/important the returned data is
- The impact of stale data being considered fresh
- How often said data will update

All of these things differ endpoint to endpoint. Balance should update very often. An immutable contract's details can be cached indefinitely.

Re the ms lib. I wouldn't recommend it for this alone, but it has many other utilities. Which is more readable?
`await delay(ms('15mins'))` vs `await delay(15 * 60 * 1000)`. I can think of many places this library would improve the code.

Reply:
This makes sense, but if it is a local constant we should camelCase them and not have them all be `STALE_TIME`?

Reply:
Updated them all to use local camelCase naming. Will add a section to coding guidelines detailing this.

Reply:
https://www.notion.so/hiropbc/Best-practices-edbb447ef17c42b59aea11b3bd4072ec#163139712e7f4295845bb757cfb28a97